### PR TITLE
[ADDED] LinkedIn Profile Scrapper

### PR DESCRIPTION
**Pull request by Mahitej28**
## Description
The changes done are as follows:
- Created LinkedIn Folder
- Added LinkedIn Profile Scraper Python Script in `linkedInspider.py`
- Created `__init__.py` file

## Resolves: #339 

This PR is created in reference to issue number #339 

- [x] Have you adhered to the repository's defined coding convention rules?
- [ ] Have you updated the 'documentation.md' file with the method/function documentation?
- [x] Have you sent a message along with the result or response?
- [x] Have you used the try-catch technique?
- [ ] Has the method/class been added to the documentation (md file)?

## Screenshots
None

## Additional Notes/Comments
None

---
I certify that I have carried out the relevant checks and provided the requisite screenshot for validation by submitting this pull request.
I appreciate your contribution.

### CODE REVIEW

File: src/scrape_up/linkedIn/linkedInspider.py [hunk 1]
```diff
@@ -0,0 +1,145 @@
...
+                experience['start_time'] = ''
+                experience['end_time'] = ''
+                experience['duration'] = ''
+            
+            item['experience'].append(experience)
+
+        
+        """
+            EDUCATION SECTION
+        """
```
Comment (nikhil25803) [inline src/scrape_up/linkedIn/linkedInspider.py:100]: Hey, the code looks good and clean, but try to implement them as class methods.
For example, 
```python
user = Linkedin(username="nikhil25803")
user.experience()
```
This should return the user's experience detail as response. Else the code is clean and good. 
Just make these changes :))

**Review by nikhil25803 — CHANGES_REQUESTED** [review]

File: src/scrape_up/linkedIn/linkedInspider.py (hunk 1 above)
Comment (Mahitej28) [reply src/scrape_up/linkedIn/linkedInspider.py:100]: Sure @nikhil25803

**Comment by Mahitej28** [comment]
@nikhil25803 do check once I have updated the LinkedIn Profile Scraper script, added a separate class for each section

**Comment by Mahitej28** [comment]
> Hey @Mahitej28 You do not have to make different class for each function, but different methods for the same class, for example you can take a look at the `github` module, in the `User` class, we do create a class of user, with username provided and create different methods for them. Check [here]
> 
> In the image below, we create a user class and then different methods of them for different functionality like followers, following, etc. (https://github.com/Clueless-Community/scrape-up/blob/main/src/scrape_up/github/users.py) image

Ok

**Comment by Mahitej28** [comment]
@nikhil25803 you can check now the code is been modified as per requirements, and methods are also being added.
Also its a humble request to add a level of PR as well to this under GSSoc ' 23

**Review by nikhil25803 — APPROVED** [review]

**Pull request merged**